.idea/misc.xml

```diff
@@ -1,5 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="FrameworkDetectionExcludesConfiguration">
+    <file type="web" url="file://$PROJECT_DIR$" />
+  </component>
```

Comment:
I wonder why this part is always being added/removed...

Reply:
I think it's because of different IDEA versions used by comitters.